test/system/030-run.bats

```diff
@@ -28,7 +28,7 @@ load helpers
 
 @test "ramalama run granite with prompt" {
     run_ramalama run --name foobar granite "How often to full moons happen"
-    is "$output" ".*Moon" "should include some info about the Moon"
+    is "$output" ".*month" "should include some info about the Moon"
```

Comment:
This seems like a test that will always be flakey... Since the models are not so deterministic...

I wonder should we check for, at least x characters used or at least x lines printed...

Reply:
I find flakey CI can really slow down development

Reply:
Yup, I did not know of a better way to test this out.

Reply:
But at least we will have tests firing now.

Reply:
If it gets flakey I guess we can try and test for x characters that are not whitespace, etc.

We must find a way of getting both the system prompt into the model "You are a helpful assistant" and the user prompt:

```
$ ./ramalama.py run granite-code hello
-world)
$ ./ramalama.py run granite-code hello
;
```

without the "You are a helpful assistant", the output is weird...